clang/include/clang/CIR/Dialect/IR/CIRTypes.h

```diff
@@ -16,6 +16,7 @@
 #include "mlir/IR/BuiltinAttributes.h"
 #include "mlir/IR/Types.h"
 #include "mlir/Interfaces/DataLayoutInterfaces.h"
+#include "clang/CIR/Dialect/IR/CIRTypes.h"
```

Comment:
Isn't this including the current file?

Reply:
Yes, I was adding it to the other file, for the test, it was working before, and I was thinking of adding a test that covers Vec FP. I will remove it and convert this PR to NFC for now, and when we need more tests, we can add